sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -30,6 +30,7 @@ public partial class EventHubOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsF
         public int BatchCheckpointFrequency { get { throw null; } set { } }
         public Azure.Messaging.EventHubs.EventHubsRetryOptions ClientRetryOptions { get { throw null; } set { } }
         public System.Uri CustomEndpointAddress { get { throw null; } set { } }
+        public bool EnableCheckpointing { get { throw null; } set { } }
```

Comment:
Is this API change expected?

Reply:
Yes.  It didn't get exported earlier due to the CI pipeline being disabled.   It's included here to generate an API View for architect review.